BaselineOfGeoJSON.package/BaselineOfGeoJSON.class/instance/baseline..st

```diff
@@ -5,10 +5,11 @@ baseline: spec
 	spec for: #common do: [
 		self 
 			neoJson: spec;
+			neoCsv: spec;
 			ogc: spec.
    		spec 
 			package: 'GeoJSON-Core' with: [
-				spec requires: #( 'NeoJSON' 'OGC')];
+				spec requires: #( 'NeoJSON' 'NeoCSV' 'OGC')];
```

Comment:
This is not a good idea. That is the usual way to bloat a libary. For most use cases you don't need csv because it is a geo json library. That needs to be done differently or the dependency need to move somewhere else